lib/vm.py

```diff
@@ -45,7 +45,7 @@ def start(self, on=None):
         return self.host.xe('vm-start', args)
 
     def shutdown(self, force=False, verify=False):
-        logging.info("Shutdown VM")
```

Comment:
same

Reply:
Necessary for understandable logs when the VM is already up and doesn't have tools (or is stuck on a PXE or grub menu). I can make it a separate commit if necessary.

```
[INFO] Shutdown VM                                                                                                                                                                                                                                                                       
[INFO] Shutdown VM
[INFO] Wait for VM halted                                                                                                                                               
```
becomes
```
[INFO] Shutdown VM                                                                                                                                                                                                                                                                       
[INFO] Shutdown VM (force)
[INFO] Wait for VM halted                                                                                                                                               
```